card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -323,7 +347,19 @@ class CardComponent private constructor(
         return configuration.isShowStorePaymentFieldEnable
     }
 
-    fun makeAddressData(outputData: CardOutputData): Address {
+    @StringRes fun getKcpBirthDateOrTaxNumberHint(input: String): Int {
+        return when {
+            input.length > KcpValidationUtils.KCP_BIRTH_DATE_LENGTH -> R.string.checkout_kcp_tax_number_hint
+            else -> R.string.checkout_kcp_birth_date_or_tax_number_hint
+        }
+    }
+
+    fun isDualBrandedFlow(cardOutputData: CardOutputData): Boolean {
+        return cardOutputData.detectedCardTypes.filter { it.isReliable }.size > 1 &&
+            cardOutputData.detectedCardTypes.filter { it.isReliable }.any { it.isSelected }
```

Comment:
Minor: can we extract `cardOutputData.detectedCardTypes.filter { it.isReliable }` to a `val` so we don't do the filtering twice?